src/test/java/tests/FilterEvalTests.java

```diff
@@ -433,6 +434,59 @@ public void created() {
     }
 
     @Test
+    public void updatedBySelfAndOther(){
+        LocalDateTime now = LocalDateTime.now();
+        Qualifier.setCurrentTime(now);
+        TurboIssue issueUpdated = new TurboIssue(REPO, 1, "");
+        IssueMetadata updatedByOtherMetadata = new IssueMetadata(issueUpdated.getMetadata(), now.minusHours(4),
+                now.minusHours(4), 2, 2, true, false); //UpdatedByOther true with updated time 4 hours ago.
+        issueUpdated.setMetadata(updatedByOtherMetadata);
+        assertEquals(false, matches("updatedByOthers:<2", issueUpdated));
```

Comment:
Wanted to check for a value less than 4. I also changed tests to include 4 as the boundary.